nodejs/week1/MealWebsite/src/backend/app.js

```diff
@@ -0,0 +1,41 @@
+const express = require("express");
+const app = express();
+const meals = require("./routes/meals.js");
+const cheapMeals = require("./routes/cheap-meals.js");
+const largeMeals = require("./routes/large-meals.js");
+const randomMeal = require("./routes/randommeal.js");
+const reservations = require("./routes/reservations.js");
+const randomReservation = require("./routes/reservation.js");
+
+app.get("/", (request, response) => {
+  response.send(`Page not found`);
+});
+
+app.get("/meals", (request, response) => {
+  response.send(meals);
+});
+
+app.get("/cheapMeals", (request, response) => {
+  response.send(cheapMeals);
+});
+
+app.get("/largeMeals", (request, response) => {
+  response.send(largeMeals);
+});
+
+app.get("/randomMeal", (request, response) => {
+  response.send(randomMeal);
+});
+
+app.get("/reservations", (request, response) => {
+  response.send(reservations);
+});
+
+app.get("/randomReservation", (request, response) => {
```

Comment:
`"/randomReservation"` ➡️ `"/reservation"` (this is the route the homework specifies)